src/lib/holocene/accordion.svelte

```diff
@@ -80,9 +80,9 @@
       {/if}
     </div>
     <p class="flex items-center font-secondary">
-      {#if error}
-        <Badge class="mr-2" type="error">{error}</Badge>
-      {/if}
+      <Badge class="mr-2 {error ? '' : 'hidden'}" type="error" role="alert">
+        {error}
+      </Badge>
```

Comment:
Yeah perhaps a discussion to have with the design team. I think the `Badge` works best if we want something inline

![Screenshot 2023-08-29 at 11 16 36 AM](https://github.com/temporalio/ui/assets/15069288/25724edd-b163-4dd5-bbc2-1a3a6b712e04)

but if we have a long error it might not be great. I'll try some things out and see if we can get some feedback.

Reply:
I'm actually going to revert these changes and leave off a `role` here. This `Badge` is currently being used in instances where the error is there when the component loads so it's not even content that's updating.

Might want to revisit how we display errors on the `Accordion` component in the future though.